lightning-invoice/src/utils.rs

```diff
@@ -522,6 +522,7 @@ fn filter_channels<L: Deref>(
 	let mut min_capacity_channel_exists = false;
```

Comment:
We should update the docs above to include this new filtering behavior, if we still would like it to describe our filtering logic in detail.